### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by johnbelosf** [forcedotcom/sfdx-scanner#899] [Announcements]
While our [documentation](https://forcedotcom.github.io/sfdx-scanner/) helps you learn Salesforce Code Analyzer, Discussions connects you with other members of our community and extends your Code Analyzer knowledge. 

We’re continually improving Code Analyzer. Give us [feedback](https://research.net/r/SalesforceCA) or share your thoughts in Discussions.

In Discussions, you can:

* Ask us questions about Code Analyzer and Graph Engine. If a discussion helps uncover a bug, create a [Bug Report](https://github.com/forcedotcom/sfdx-scanner/issues/new?assignees=&labels=&template=bug_report.md&title=%5BBUG%5D).
* Share ideas with us–and other community members–on new features that you’d like to see. Once a feature request has reached a meaningful shape, create a [Feature Request](https://github.com/forcedotcom/sfdx-scanner/issues/new?assignees=&labels=USER+STORY&template=feature_request.md&title=%5BFeature+Request%5D).

To get started with Discussions, use comments to introduce yourself to the community.

**Comment by jefersonchaves** [comment]
Hello @johnbelosf - Thanks for opening this channel. My name is Jeferson Chaves, and I'm a Technical Architect at Waeg, an IBM Company.

**Comment by stephen-carter-at-sf** [comment]
Closing older discussions now that we have gone GA with code-analyzer. See https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/get-started.html for the latest and greatest.
